src/eventSubscription/createWizard/EventHubsNamespaceListStep.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EventHubManagementClient } from 'azure-arm-eventhub';
+import { EHNamespace, Eventhub } from 'azure-arm-eventhub/lib/models';
+import { AzureWizardPromptStep, IAzureQuickPickItem, IAzureQuickPickOptions, IAzureUserInput, ISubscriptionWizardContext } from 'vscode-azureextensionui';
+import { localize } from '../../utils/localize';
+
+export interface IEventHubsNamespaceWizardContext extends ISubscriptionWizardContext {
+    eventHubsNamespace?: EHNamespace;
+}
+
+export class EventHubsNamespaceListStep<T extends IEventHubsNamespaceWizardContext> extends AzureWizardPromptStep<T> {
+    public async prompt(wizardContext: T, ui: IAzureUserInput): Promise<T> {
+        if (!wizardContext.eventHubsNamespace) {
+            const quickPickOptions: IAzureQuickPickOptions = { placeHolder: localize('listPlaceHolder', 'Select an Event Hubs Namespace'), id: `EventHubsNamespaceListStep/${wizardContext.subscriptionId}` };
```

Comment:
nit: "Event Hubs namespace"?

Reply:
Lol I think it's weird, too, but that's what the portal and documentation uses
![screen shot 2018-05-18 at 9 27 39 am](https://user-images.githubusercontent.com/11282622/40246493-caf44d4a-5a7d-11e8-8948-8da493d94e0a.png)
![screen shot 2018-05-18 at 9 27 47 am](https://user-images.githubusercontent.com/11282622/40246494-cc49f2d0-5a7d-11e8-9184-810f9fd2a01d.png)